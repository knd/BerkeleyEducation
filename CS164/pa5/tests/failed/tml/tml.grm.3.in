<tag1 attr1="val1 val1.1">
	<tag2 attr2="val2" attr2_2="val2">
		<tag3>  </tag3>
	</tag2>
</tag1>
