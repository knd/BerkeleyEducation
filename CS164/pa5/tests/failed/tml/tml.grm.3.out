{'children': {0: {'attr2_2': 'val2', 'attr2': 'val2', 'name': 'tag2', 'children': {0: {'name': 'tag3', 'children': {}}}}}, 'name': 'tag1', 'attr1': 'val1 val1.1'}
